test/tests/functional/pbs_snapshot_unittest.py

```diff
@@ -348,7 +350,7 @@ def test_capture_pbs_logs(self):
             self.skipTest("No PBSPro daemons found on the system," +
                           " skipping the test")
 
-        with PBSSnapUtils(out_dir=target_dir, acct_logs=num_acct_logs,
+        with PBSSnapUtils(out_dir=self.parent_dir, acct_logs=num_acct_logs,
```

Comment:
Could this cause a problem when running on an NFS mounted volume that squashes root? We might want to test for this early on.

Reply:
Shouldn't be, pbs_snapshot should just move on if it can't capture some of the information (for whatever reason, root squashing included), it is designed that way. If it breaks, then that's a bug that we should fix. I'm not sure that I can write a PTL test for this though.